File: src/org/blitzortung/android/map/overlay/OwnLocationOverlayItem.java
package org.blitzortung.android.map.overlay;

import org.blitzortung.android.data.Coordsys;

import android.location.Location;

import com.google.android.maps.OverlayItem;

public class OwnLocationOverlayItem extends OverlayItem {

	private float radius;
	
	public OwnLocationOverlayItem(Location location, float radius) {
		super(Coordsys.toMapCoords((float)location.getLongitude(), (float)location.getLatitude()), "", "");
		
		this.radius = radius;
	}
	
	public float getRadius() {
		return radius;
	}

}

File: app/src/main/java/org/blitzortung/android/map/overlay/OwnLocationOverlayItem.kt
package org.blitzortung.android.map.overlay

import android.location.Location

import com.google.android.maps.OverlayItem

import org.blitzortung.android.data.Coordsys

class OwnLocationOverlayItem(location: Location, val radius: Float) : OverlayItem(Coordsys.toMapCoords(location.longitude.toFloat(), location.latitude.toFloat()), "", "")
